test/integration/targets/vault/runme.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 
 set -euvx
+source virtualenv.sh
 
+source virtualenv.sh
```

Comment:
You only need to invoke `source virtualenv.sh` once.

Reply:
Sorry, this is auto commit by github-review.  I'll delete it.

Reply:
done.